apps/dashboard/lib/get-balance-for-transactions.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  TransactionEdge,
+  TxDirection,
+  WalletCurrency,
+} from "@/services/graphql/generated"
+import { ProcessedTransaction } from "./index.types"
+import { formatDateTime, formatMonth } from "@/app/utils"
+
+export const getBalanceForTransactions = ({
```

Comment:
the name is not great. IIUC, it should be more `getMinMaxBalanceAndPerWalletTransaction`